package.json

```diff
@@ -24,6 +24,7 @@
     "format": "prettier --write 'src/**/*.{js,ts}'"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.5",
```

Comment:
Is this going to be needed at runtime? We distribute a production npm package of this -- I had to add `@babel/runtime` to get that working a little ways back.